docs/kubernetes/kctlr-app-install.rst

```diff
@@ -1,21 +1,10 @@
-.. STAGED IN K8S_BIGIP_CTLR REPO
-.. todo:: remove from this repo and add redirect in AWS
-
-.. index::
-   single: BIG-IP Controller; Kubernetes
-
-.. sidebar:: Docs test matrix
-
-   Documentation manually tested with:
-
-   - kubernetes-v1.6.4 on Ubuntu-16.4.2
-   - kubernetes-v1.4.8 on CoreOS 1409.6.0
-   - ``k8s-bigip-ctlr`` v1.0.0-v1.4.0
+:product: BIG-IP Controller for Kubernetes
+:type: task
 
 .. _install-kctlr:
 
-Install the BIG-IP Controller in Kubernetes
-===========================================
+Install the BIG-IP Controller: Kubernetes
+=========================================
 
 The |kctlr-long| installs via a Kubernetes `Deployment`_.
 The Deployment creates a `ReplicaSet`_ that, in turn, launches a `Pod`_ running the |kctlr| app.
```

Comment:
Do we mention anywhere that it's important to not have duplicate controllers running? That is to say, the `replicas` field should always be 1 (or 0 if you want to remove the controller temporarily).

Reply:
Good catch. Adding that in.